data/campaigns/Son_Of_The_Black_Eye/scenarios/04_The_Siege_of_Barag_Gor.cfg

```diff
@@ -358,22 +341,38 @@
             message= _ "KILL HIM!!"
         [/message]
 
+        [move_unit_fake]
+            side=5
+            type=Elvish Fighter
+            x=28,29
+            y= 6, 6
+        [/move_unit_fake]
+
         [unit]
```

Comment:
Would it be possible to use random_gender=yes here?
Maybe by first spawning the unit and then moving it with [move_unit]

Reply:
Elvish fighters have one gender only at the moment

Reply:
true…